File: LojaMargun-Domain/Entities/Product.cs
namespace LojaMargun_Domain.Entities
{
    public class Product : EntityBase
    {
        public int ItemId { get; set; }
        public Item Item { get; set; }
        public int CategoryId { get; set; }
        public virtual Category Category { get; set; }
        public int Quantity { get; set; }
        public bool Active { get; set; }
    }

}

File: LojaMargun-API/ClientApp/src/app/Models/Product.ts
import { Category } from "./Category";

export class Product{
   id!: number;
   name!: string;
   description!: string;
   length!: string;
   image!: string;
   value!: number;
   quantity!: number;
   categoryId!: number;
   category!: Category;
   active!: boolean;
}